### PR TITLE
Add global params for limit and offset

### PR DESCRIPTION
**Pull request by reivaj05**

### CODE REVIEW

File: .github/workflows/jobs.yaml [hunk 1]
```diff
@@ -1169,7 +1169,7 @@ jobs:
           go tool cover -func=all.out | grep total > tmp2
           result=`cat tmp2 | awk 'END {print $3}'`
           result=${result%\%}
-          threshold=71.4
+          threshold=71.3
```
Comment (reivaj05) [inline .github/workflows/jobs.yaml:1172]: Reducing threshold because for some reason https://github.com/minio/console/pull/3094/ decreased it

**Review by cesnietor — DISMISSED** [review]
LGTM

File: restapi/ws_handle.go [hunk 2]
```diff
@@ -164,11 +164,11 @@ func serveWS(w http.ResponseWriter, req *http.Request) {
 	// If we are using a subpath we are most likely behind a reverse proxy so we most likely
 	// can't validate the proper Origin since we don't know the source domain, so we are going
 	// to allow the connection to be upgraded in this case.
-	if getSubPath() != "/" || getConsoleDevMode() {
-		upgrader.CheckOrigin = func(r *http.Request) bool {
-			return true
-		}
+	// if getSubPath() != "/" || getConsoleDevMode() {
```
Comment (bexsoft) [inline restapi/ws_handle.go]: Is it necessary to disable this line?

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: restapi/ws_handle.go (hunk 2 above)
Comment (reivaj05) [reply restapi/ws_handle.go]: my bad, i just added the all directory to git

Comment (cesnietor) [reply restapi/ws_handle.go]: good catch

**Review by cesnietor — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**